handbook/product/surfacing_product_feedback.md

```diff
@@ -0,0 +1,30 @@
+# Surfacing feedback to the product team
+
+> If you're working with a specific customer on a bug or issue and need to directly involve a product engineering team, please refer to [engaging other teams](../ce/engaging-other-teams.md). 
+
+CEs, CSEs, sales, marketing, engineering, and other teammates that interact directly with Sourcegraph users and/or Sourcegraph itself should share product feedback with the product team. 
+
+We deeply value this feedback, so we make this process as frictionless for teammates as possible: 
+
+## Post the feedback in [`#feedback`](https://sourcegraph.slack.com/archives/C0W2E592M) on Slack
+
+You're welcome to tag teammates or teams, but that's not necessary – once the feedback is in [#feedback](https://sourcegraph.slack.com/archives/C0W2E592M) the product team will make sure it's routed correctly. 
+
+## When to create a GitHub issue
+
+If you have gotten a specific feedback requests that a customer(s) would like to track the progress of and a GitHub issue for the feedback [does not already exist](https://github.com/sourcegraph/sourcegraph/issues), you can also document this in a [new GitHub issue](https://github.com/sourcegraph/sourcegraph/issues/new/choose) to share with the customer. 
+
+Because a shareable issue is by definition public, you should not include any private customer information. You should still point to the customer by including a link to the customer in Salesforce.
+
+After you create a GitHub issue to share with a customer, please note in the body of the issue that `This issue was created so that a customer can track its progress` so the team is aware. Then you should share a link to this issue in `#feedback`. 
```

Comment:
@Joelkw We sometimes do make sourcegraph/customers issues with more specific info if needed, and link to them in the public issue. Is that still cool?

Reply:
(Also can we make an issue template with that line included + an inline warning not to include customer info + whatever labels you want? That might help for data standardization.)

Reply:
Haha yeah, that's still cool! (Haha because I originally had an explicit line about that here but took it out bc I thought it might be unnecessary given the note on L3. But I'll add it back in and create an issue template here.)